signer/tests/integration/postgres.rs

```diff
@@ -1183,8 +1183,8 @@ async fn writing_completed_deposit_requests_postgres() {
 
     let (txid, block_id, amount, bitcoin_txid, vout) = db_event.pop().unwrap();
 
-    assert_eq!(txid, event.txid.0);
-    assert_eq!(block_id, event.block_id.0);
+    assert_eq!(txid, event.txid.0 .0);
+    assert_eq!(block_id, event.block_id.0 .0);
```

Comment:
For this, have the query deserialize to a `BitcoinTxId` and a `BitcoinBlockHash` to make this comparison easy here. Same thing for the other tests.